defaults/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+ovirt_engine_setup_dwh: True
+ovirt_engine_setup_version: '4.1'
+
+ovirt_engine_setup_provider_ovn_username: 'admin@internal'
+
+ovirt_engine_setup_db_host: 'localhost'
+ovirt_engine_setup_db_port: 5432
+ovirt_engine_setup_db_name: 'engine'
+ovirt_engine_setup_db_user: 'engine'
+
+ovirt_engine_setup_dwh_db_configure: true
+# ^ set this to false if you want to install the dwh on a different host
+# machine rather than the same machine where the engine is installed.
+ovirt_engine_setup_dwh_db_host: 'localhost'
+ovirt_engine_setup_dwh_db_port: 5432
+ovirt_engine_setup_dwh_db_name: 'ovirt_engine_history'
+ovirt_engine_setup_dwh_db_user: 'ovirt_engine_history'
+
+ovirt_engine_setup_configure_iso_domain: false
+ovirt_engine_setup_iso_domain_path: '/var/lib/exports/iso'
+ovirt_engine_setup_iso_domain_name: 'ISO_DOMAIN'
+ovirt_engine_setup_iso_domain_acl: '0.0.0.0/0.0.0.0(rw)'
```

Comment:
ISO is no longer valid for 4.2, please remove those options from default

Reply:
Done